config/config.toml

```diff
@@ -19,10 +19,11 @@ allow_no_verification=true
 # disable_verification=false
 # Causes TestRunWithDefaultConfig to break, but
 # fine to use in /etc/soci-snapshotter-grpc-config.toml
-max_concurrency=0  # Actually zero
+max_concurrency=0
 no_prometheus=false
 mount_timeout_sec=0
 fuse_metrics_emit_wait_duration_sec=0
+force_get_xattrs=false
```

Comment:
this config name is a bit weird. why just call it "disable_xattrs" and default to "true"?

Reply:
I think `disable_xattrs` is a bit of a misleading name. The ultimate goal of the flag is to decide whether or not the runtime honors the DisableXAttrs annotation. The variable names were intentionally chosen to avoid making users believe this disables xattrs for files that have xattrs, when it reality it just skips the check for layers that don't have them anyway.

With that in mind, I think the variable name is fine, but I'm open for alternative variable names.

Reply:
The DisableXAttrs annotation was set internally based on "optimizations" list. This PR changed the logic based on a new config/flag. Either way it's transparent to end users, who seem only care about if [DisableXAttrs](https://github.com/awslabs/soci-snapshotter/blob/main/fs/fs.go#L604C3-L604C16) FUSE mount option is turned on.

Another option here is enable all optimizations and allow users to opt-out individual ones, and change the optimizations flag to something like "disable-optimizations".

Reply:
> Either way it's transparent to end users, who seem only care about if [DisableXAttrs](https://github.com/awslabs/soci-snapshotter/blob/main/fs/fs.go#L604C3-L604C16) FUSE mount option is turned on.

It doesn't inherently turn it on, though, since it only means we'll read the annotation.

With this comment + previous comments on when I plumbed this variable into the code, it's becoming pretty clear that my verbosity in clarifying variable behavior is probably doing the opposite, so I'll just keep it simple here and use `disable_xattrs`.